drivers/npmx/npmx_shell.c

```diff
@@ -3310,6 +3310,17 @@ static int timer_config_set(const struct shell *shell, size_t argc, char **argv,
 		shell_error(shell, "Error: unable to set timer configuration.");
 	}
 
+	if (config_type == TIMER_CONFIG_TYPE_COMPARE) {
+		/* Period timer needs strobe to update */
+		err_code = npmx_timer_task_trigger(timer_instance, NPMX_TIMER_TASK_STROBE);
+
+		if (check_error_code(shell, err_code)) {
+			shell_print(shell, "Success: timer updated.");
+		} else {
+			shell_error(shell, "Error: unable to update timer configuration.");
+		}
```

Comment:
If timer strobe is related only with compare value, then this section should be moved after line 3306 inside `else if (config_type == TIMER_CONFIG_TYPE_COMPARE) {` . `Success` message should be printed only once as `shell_print(shell, "Success: %d.", timer_config.compare_value);` also.

Reply:
Description of TIMERTARGETSTROBE register suggests that strobe is only for compare_value/period/timer value:
https://infocenter.nordicsemi.com/topic/ps_npm1300/chapters/core_components/timer/doc/frontpage.html?cp=7_1_0_6_2_5_2#register.TIMERTARGETSTROBE

Reply:
I agree with @jaz1-nordic - `success` should be printed only once.